.github/config.yml

```diff
@@ -51,6 +51,7 @@ checks:
   mobile-compile-time-options:
     name: Mobile/Compile time options
     on-run:
+    - mobile-compile-time-cc
```

Comment:
sorry which trigger paths ?

the request job reads the config and generates the ci data for the other jobs - including what to run

Reply:
there is a fuller explanation of how the ci works here https://github.com/envoyproxy/envoy/blob/main/.github/workflows/README.md

Reply:
the explanation is in the link posted above

Reply:
I read the link posted above and it wasn't clear to me.  It's why I was asking for clarification

Reply:
essentially - the `request` job starts, reads the configuration, and determines what should/not run

according to what should run it also starts or skips `checks`

the rest of the workflows load the `request` data and determine whether any of the jobs under the wf need to run (amongst  other things) and at the end of the wf fail or pass the checks

Reply:
Ok I think it'd be much more clear if we simply didn't reuse strings between runs and checks.  is that something you could do as a follow-up as I think it would have avoided the confusion more than the docs do alone

Reply:
yeah, this makes sense - agreed it can be confusing if they are named the same